build/gulpfile.extensions.js

```diff
@@ -40,23 +40,23 @@ const tasks = compilations.map(function (tsconfigFile) {
 	const tsOptions = _.assign({}, tsconfig.extends ? require(path.join(extensionsPath, relativeDirname, tsconfig.extends)).compilerOptions : {}, tsconfig.compilerOptions);
 	tsOptions.verbose = false;
 	tsOptions.sourceMap = true;
+	const outDir = tsOptions.outDir || 'out';
 
 	const name = relativeDirname.replace(/\//g, '-');
 
 	const root = path.join('extensions', relativeDirname);
 	const srcBase = path.join(root, 'src');
 	const src = path.join(srcBase, '**');
-	const out = path.join(root, 'out');
+	const out = path.join(root, outDir);
```

Comment:
This was a bug fix to honor the outDir setting in the tsconfig.json of every extension project.